### PR TITLE
Add a very rough outline for a spec.

### PR DESCRIPTION
**Pull request by mkruisselbrink**
Doesn't actually specify any behavior yet, but is at least a start.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+Group: WICG
+Repository: wicg/writable-files
+Indent: 2
+Complain About: accidental-2119 yes, missing-example-ids yes
+Markup Shorthands: css no, markdown yes
+</pre>
+
+<pre class=link-defaults>
+spec:fetch; type:interface; text:ReadableStream
+spec:infra; type:dfn; for:/; text:set
```
Comment (inexorabletash) [inline index.bs]: You should be able drop this if you use `[=/set=]` (at least, I was able to in one of my specs)

File: index.bs [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+</div>
+
+### The {{FileSystemHandle/copyTo()}} method ### {#api-filesystemhandle-copyto}
+
+<div class=note><dl class=domintro>
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|copyTo}}(await
+        |handle|.{{FileSystemHandle/getParent()}}, |newName|)
+  :: Creates a copy of the entry represented by |handle| with name |newName|.
+
+  : |newHandle| = await |handle| . {{FileSystemHandle/moveTo()|moveTo}}(|otherDir|)
```
Comment (inexorabletash) [inline index.bs]: nit: s/moveTo/copyTo/ here and in the next dt

File: index.bs [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+<pre class=link-defaults>
+spec:fetch; type:interface; text:ReadableStream
+spec:infra; type:dfn; for:/; text:set
+</pre>
+
+<pre class=anchors>
+</pre>
+
+<style>
+dl.domintro dt {
```
Comment (inexorabletash) [inline index.bs]: I know you're replicating what other specs (mine?) do, but if we changed these selectors from `dl.domintro` to just `.domintro` then we could use `<div class="note domintro">` below and avoid the explicit `<dl>` tag.

File: index.bs [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+
+# Accessing native filesystem # {#native-filesystem}
+
+## The {{Window/chooseFileSystemEntries()}} method ## {#api-choosefilesystementries}
+
+<xmp class=idl>
+enum ChooseFileSystemEntriesType { "openFile", "saveFile", "openDirectory" };
+
+dictionary ChooseFileSystemEntriesOptionsAccepts {
+  DOMString description;
```
Comment (inexorabletash) [inline index.bs]: Make these `USVString`s, since they're probably going to get converted to UTF-8 at some point.

File: index.bs [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+enum SystemDirectoryType {
+  "sandbox"
+};
+
+dictionary GetSystemDirectoryOptions {
+  required SystemDirectoryType type;
+};
+
+partial interface FileSystemDirectoryHandle {
+  static Promise<FileSystemDirectoryHandle> getSystemDirectory(GetSystemDirectoryOptions options);
```
Comment (inexorabletash) [inline index.bs]: Hmmm... except for `sandbox` ISTM we'd want to hang this off Window instead, so operating systems can prompt the user (for permission, fallback directory, etc). Issue to track this?

File: index.bs (hunk 1 above)
Comment (mkruisselbrink) [reply index.bs]: Ah yes, I thought I tried that but must not have (and must have gotten confused by bikeshed showing multiple options for `/set` when I was just using `[=set=]`. But I guess the other one isn't exported so doesn't get picked up. Done.

File: index.bs (hunk 3 above)
Comment (mkruisselbrink) [reply index.bs]: Ah yes, that's nicer. Done.

File: index.bs (hunk 2 above)
Comment (mkruisselbrink) [reply index.bs]: Done

File: index.bs (hunk 4 above)
Comment (mkruisselbrink) [reply index.bs]: Good point. Done.

File: index.bs (hunk 5 above)
Comment (mkruisselbrink) [reply index.bs]: Yeah, I don't really have strong opinions on how this is exposed. Linked to the naming bikeshedding issue, as well as mentioned it perhaps being on Window. Although I don't see how the interface it is exposed on changes what operating systems can do wrt prompting?

Comment (inexorabletash) [reply index.bs]: Just thinking if it's on Window then it's more obvious that you can't use it from serviceworkers where you can't prompt. But we need to solve that eventually and exceptions are a thing, so yeah - not a big reason to put it on Window.

**Pull request merged**